scene/2d/ray_cast_2d.cpp

```diff
@@ -267,10 +267,10 @@ void RayCast2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_collision_normal"), &RayCast2D::get_collision_normal);
 
 	ClassDB::bind_method(D_METHOD("add_exception_rid", "rid"), &RayCast2D::add_exception_rid);
-	ClassDB::bind_method(D_METHOD("add_exception", "node"), &RayCast2D::add_exception);
+	ClassDB::bind_method(D_METHOD("add_exception", "node:CollisionObject2D"), &RayCast2D::add_exception);
```

Comment:
Not sure if those would be ok. The method clearly excepts a CollisionObject2D, but allows you to pass any type of Object.